pkg/addons/assets/efa-device-plugin.yaml

```
@@ -56,6 +56,7 @@ spec:
                       - g4dn.8xlarge
                       - g4dn.metal
                       - g5.48xlarge
+                      - g6.48xlarge
```

This list needs to be updated with all G6 instance types. 
```suggestion
                      - g6.8xlarge
                      - g6.12xlarge
                      - g6.16xlarge
                      - g6.24xlarge
                      - g6.48xlarge
```